### PR TITLE
Test to run criu test suit - v2

### PR DESCRIPTION
**Pull request by Pavithra1602**
This script downloads criu tarball and executes zdtm tests
Version can be specified in version.yaml file

Signed-off-by: pavrampu@linux.vnet.ibm.com <pavrampu@in.ibm.com>

### CODE REVIEW

File: generic/criu.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        self.srcdir = os.path.join(self.srcdir, "test")
+        build.make(self.srcdir, ignore_status=True)
+
+    def test(self):
+        os.chdir(self.srcdir)
+        process.run("./zdtm.py run -a --report sergeyb --keep-going", ignore_status=True, sudo=True)
+        logfile = os.path.join(self.logdir, "stdout")
+        failed_tests = process.system_output("grep -w FAIL %s" % logfile, shell=True, ignore_status=True)
+        if failed_tests:
+            self.fail("few tests failed")
```
Comment (PraveenPenguin) [inline generic/criu.py]: can we also show failed test cases ?
```

        if failed_tests:
            self.log.info(failed_tests)
            self.fail("few tests failed")
```

File: generic/criu.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+from avocado.utils import process
+from avocado.utils.software_manager import SoftwareManager
+
+
+class CRIU(Test):
+
+    def setUp(self):
+        sm = SoftwareManager()
+        dist = distro.detect()
+        packages = ['gcc', 'make', 'protobuf', 'protobuf-c', 'protobuf-c-devel', 'protobuf-compiler', 'protobuf-devel', 'protobuf-python', 'libnl3-devel', 'libcap-devel', 'libaio-devel']
```
Comment (narasimhan-v) [inline generic/criu.py]: line length, please

Comment (PraveenPenguin) [inline generic/criu.py]: @narasimhan-v  thanks , please wrap this in 80 char

**Comment by PraveenPenguin** [comment]
closing this PR and new version of this test raised #93

**Pull request closed without merging**